public/js/log.js

```diff
@@ -60,250 +59,295 @@ MAGI_log.resizes = {
 };
 MAGI_log.tooltips = [];
 
-$('#magi-loggingReadMore').click(function(e) {
-  e.preventDefault();
-  var isVisible = 'visible' == $('#magi-loggingReadMoreDiv').css('visibility'),
-      visibleState = isVisible ? 'hidden' : 'visible',
-      displayState = isVisible ? 'none' : 'block';
-
-  $('#magi-loggingReadMoreDiv').css('visibility', visibleState);
-  $('#magi-loggingReadMoreDiv').css('display', displayState);
-
+$().ready(function () {
+  $.get('/logEnabled', function(res, status) {
```

Comment:
Would you be fine with me merging this as-is and create an issue for adding
this, as I don't think this is a blocking issue? I'm not going to be able
to hack this week and dhl wanted the revised logging up ASAP.

On Mon, May 11, 2015 at 10:09 PM Max Leiserson <notifications@github.com>
wrote:

> In public/js/log.js
> <https://github.com/raphael-group/magi/pull/4#discussion_r30099878>:
>
> > @@ -60,250 +59,295 @@ MAGI_log.resizes = {
> >  };
> >  MAGI_log.tooltips = [];
> >
> > -$('#magi-loggingReadMore').click(function(e) {
> > -  e.preventDefault();
> > -  var isVisible = 'visible' == $('#magi-loggingReadMoreDiv').css('visibility'),
> > -      visibleState = isVisible ? 'hidden' : 'visible',
> > -      displayState = isVisible ? 'none' : 'block';
> > -
> > -  $('#magi-loggingReadMoreDiv').css('visibility', visibleState);
> > -  $('#magi-loggingReadMoreDiv').css('display', displayState);
> > -
> > +$().ready(function () {
> > +  $.get('/logEnabled', function(res, status) {
>
> Okay that sounds good, but I still think it makes more sense to pass a
> variable from the server. Then you don't even have to load the Javascript
> files, and it should be pretty easy to implement. I don't think you'd have
> to change any of log.js.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/raphael-group/magi/pull/4/files#r30099878>.
>